esphome/components/mr24d11c10/__init__.py

```diff
@@ -0,0 +1,37 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import uart
+from esphome.const import CONF_ID
+
+DEPENDENCIES = ["uart"]
+CODEOWNERS = ["@TheRealTom"]
+MULTI_CONF = True
+
+mr24d11c10_ns = cg.esphome_ns.namespace("mr24d11c10")
+MR24D11C10Component = mr24d11c10_ns.class_(
+    "MR24D11C10Component", cg.Component, uart.UARTDevice
+)
+CONF_MR24D11C10_ID = "mr24d11c10_id"
+
+CONFIG_SCHEMA = cv.All(
```

Comment:
`cv.All` is not required here
```suggestion
CONFIG_SCHEMA = (
```